py/visdom/server.py

```diff
@@ -295,6 +299,9 @@ def __init__(self, *request, **kwargs):
         self.include_host = True
         super(BaseHandler, self).__init__(*request, **kwargs)
 
+    def get_current_user(self):
```

Comment:
Add a docstring here, as this isn't referenced anywhere else.

In combination with self.current_user I'm getting the idea that tornado calls this function to populate `current_user`, but that's unclear from the code.